### DISCUSSION TITLE
Add HEIF and HEVC to allowedFileExtensions

### DISCUSSION THREAD

**Discussion by mariovass** [craftcms/cms#12490] [Ideas] [answered]
Would you be able to add HEIF and HEVC file types to allowedFileExtensions in Assets please ? Customers uploading photos from iphone are generally in these file formats.

**Comment by brianjhanson** [comment]
You can add `.heif` and `.hevc` to the `allowedFileExtensions` array via the [`extraAllowedFileExtensions`](https://craftcms.com/docs/4.x/config/general.html#extraallowedfileextensions) config setting. 

You'll also probably want to update the [`extraFileKinds`](https://craftcms.com/docs/4.x/config/general.html#extrafilekinds) config setting if you have any file type restrictions on your asset fields. 

Do you have a use case where the config is falling short?

**Comment by brandonkelly** [comment, marked as answer]
The next Craft 3 and 4 releases will allow uploading `heic`, `heif`, and `hevc` files by default. HEIC image manipulation support has also been added, for servers that support it. (d7ee2e20b291b055b3ef8a93cf31da1800ba6103, #9115)

**Reply by brandonkelly** [reply]
Craft 3.7.63 and 4.3.6 are out with those changes!
